City-of-Helsinki/drupal-helfi-platform-config
# UHF-919: Sitemap feature

To test this:

- `composer require drupal/helfi_platform_config:dev-UHF-919`
- Revert helfi_base_feature

Go to Structure -> Content types -> Edit some content type and make sure that sitemap feature can be enabled for selected content type
